GitVersion.yml

```diff
@@ -0,0 +1,5 @@
+mode: Mainline
+next-version: 3.1.0
+branches:
+   master:
+      regex: ^master$|^main$
```

Comment:
why is it restricted to master and main????

Reply:
regex expression in the Gitversion.yml file only work on the identified branch
so gitversion can properly identify the branch type that's why we use master and main only for release part